server/scripts/fig_wrapper.sh

```diff
@@ -2,23 +2,27 @@
 
 TIMEOUT=2
 
+MONGO_HOSTNAME=${MONGO_HOSTNAME:="mongodb"}
```

Comment:
it would be better to use `MONGO_URI`, `ELASTICSEARCH_URL` and `REDIS_URL` env vars, those are used by the app and have port number in it, so it should work on different ports as well